base/bucket_gocb.go

```diff
@@ -119,6 +119,10 @@ func QueryBucketItemCount(ctx context.Context, n1qlStore N1QLStore) (itemCount i
 	return val.Count, nil
 }
 
+func IsMinimumVersion(major, minor, requiredMajor, requiredMinor uint64) bool {
+	return isMinimumVersion(major, minor, requiredMajor, requiredMinor)
+}
+
 func isMinimumVersion(major, minor, requiredMajor, requiredMinor uint64) bool {
```

Comment:
```suggestion
func IsMinimumVersion(major, minor, requiredMajor, requiredMinor uint64) bool {
```

Is fine, and probably you will have to change the test code to use exported function.

Reply:
got it will change it